src/cartridges/int_adyen_overlay/cartridge/scripts/util/adyenHelper.js

```diff
@@ -775,12 +775,6 @@ var adyenHelperObj = {
       integrator: this.getSystemIntegratorName(),
     };
 
-    if (isEcom) {
```

Comment:
We should now also remove the `isEcom` variable from the function `getApplicationInfo(isEcom)`